main.tf

```diff
@@ -0,0 +1,213 @@
+data "aws_vpc" "shared" {
+  tags = {
+    "Name" = var.vpc_all
+  }
+}
+
+data "aws_kms_key" "s3" {
```

Comment:
Should this key be hard-coded? I'd have thought a data call would be a better way of finding the correct key here.